web/src/TabNav.tsx

```diff
@@ -86,26 +86,15 @@ export function LegacyNavProvider(
 
 // New Overview semantics:
 //
-// 1. When you single click a resource on the left sidebar,
+// 1) When you click or ctrl-click a resource on the left sidebar that's already open,
 //    it changes the current tab to the new resource.
 //
-// 2. When you ctrl-click a resource on the left sidebar,
-//    it opens a new tab, and brings the new tab into focus.
+// 2) Otherwise, when you select a resource on the left sidebar,
+//    a) click opens it in the current tab,
+//    b) ctrl-click opens it in a new tab on the immediate right of current tab
```

Comment:
Not part of this PR, but I noticed during review:

For me on Firefox, the Mac OS context menu still has precedence. I assume the expected behavior is that Tilt should hijack?

(Maybe I'm missing something. You're able to override the context menu on Mac?)

Reply:
yep, on macos it's the command key

fwiw, most of the standards docs on this tend to be super vague and hand-wavey for exactly the reason you describe - this tends to be dictated by a complicated set of overlapping OS-level preferences and browser-level preferences, so i changed this comment to be more hand-wavey